subsys/net/lib/dhcpv4/Kconfig

```diff
@@ -44,6 +44,22 @@ config NET_DHCPV4_ACCEPT_UNICAST
 	  If set, the network stack will accept unicast DHCPv4 responses from
```

Comment:
Could you modify the commit subject, the option 60 does not tell anything and no need to mention dhcpv4 twice.
Perhaps something like `net: dhcpv4: Add vendor class identifier option`